topdown/encoding.go

```diff
@@ -258,6 +267,31 @@ func builtinYAMLIsValid(a ast.Value) (ast.Value, error) {
 	return ast.Boolean(err == nil), nil
 }
 
+func builtinHexEncode(a ast.Value) (ast.Value, error) {
+	str, err := builtins.StringOperand(a, 1)
+	if err != nil {
+		return nil, err
+	}
+	src := []byte(str)
+	dst := make([]byte, hex.EncodedLen(len(src)))
+	hex.Encode(dst, src)
+	return ast.String(dst), err
+}
+
+func builtinHexDecode(a ast.Value) (ast.Value, error) {
+	str, err := builtins.StringOperand(a, 1)
+	if err != nil {
+		return nil, err
+	}
+	src := []byte(str)
+	dst := make([]byte, hex.DecodedLen(len(src)))
+	_, err = hex.Decode(dst, src)
```

Comment:
Similarly, could we use [`hex.DecodeString`](https://golang.org/pkg/encoding/hex/#DecodeString)?

Reply:
Yes, I've updated.